src/test/java/com/adobe/cq/commerce/graphql/client/impl/GraphqlClientAdapterFactoryTest.java

```diff
@@ -15,20 +15,44 @@
 package com.adobe.cq.commerce.graphql.client.impl;
 
 import org.apache.sling.api.resource.Resource;
+import org.apache.sling.testing.mock.sling.ResourceResolverType;
 import org.junit.Assert;
+import org.junit.Before;
 import org.junit.Rule;
 import org.junit.Test;
 
 import com.adobe.cq.commerce.graphql.client.GraphqlClient;
 import io.wcm.testing.mock.aem.junit.AemContext;
+import io.wcm.testing.mock.aem.junit.AemContextBuilder;
 
+import static org.apache.sling.testing.mock.caconfig.ContextPlugins.CACONFIG;
 import static org.mockito.Mockito.mock;
 import static org.mockito.Mockito.when;
 
 public class GraphqlClientAdapterFactoryTest {
 
+    public final static String CATALOG_IDENTIFIER = "my-catalog";
+
     @Rule
-    public final AemContext context = GraphqlAemContext.createContext("/context/graphql-client-adapter-factory-context.json");
+    public final AemContext context = new AemContextBuilder(ResourceResolverType.JCR_MOCK).plugin(CACONFIG).build();
+
+    Resource mockConfigurationResource;
+
+    @Before
+    public void setUp() {
```

Comment:
Why duplicate that code from `GraphqlAemContext` here? Better adapt ``GraphqlAemContext` to be able to call the `plugin()` method, otherwise you might as well remove `GraphqlAemContext` because it was only used for this test.